src/layout/Footer.tsx

```diff
@@ -34,6 +35,9 @@ export const Footer = () => {
         bgcolor: 'info.light',
         color: 'primary.main',
       }}>
+      <Typography component="h2" sx={visuallyHidden}>
```

Comment:
Etter en sjekk av nav.no, regjeringen.no og uutilsynet.no ser det ut som normen egentlig er å ha bare headingene i footer rett under det som er parent på siden over uten å legge inn en egen (skjult) "hovedheading" i footeren. Mulig det egentlig er best-practice da, men vet ikke. Opp til deg!

Reply:
Sjekke litt mer rundt, og fant ulike argument for både for og i mot, men tror nok jeg er enig med deg her. Lar `<footer>` ta seg av jobben, som virker som om mange brukere er fornøyd med.